corehq/apps/users/management/commands/populate_user_role.py

```diff
@@ -21,6 +25,48 @@ def sql_class(self):
     def commit_adding_migration(cls):
         return "TODO: add once the PR adding this file is merged"
 
+    @classmethod
+    def diff_couch_and_sql(cls, couch, sql):
+        diffs = []
+        for field in UserRole._migration_get_fields():
+            diffs.append(cls.diff_attr(field, couch, sql))
+
+        couch_upstream_id = couch["upstream_id"]
```

Comment:
`couch` is just a plain dict, isn't it, that has never been wrapped? `upstream_id` is a pretty recent addition that I'd expect to be missing from older docs.

Reply:
a4d42797433